src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_NUMBER = 10;
+    private K[] keysStorage;
+    private V[] valuesStorage;
+
+    private int size = 0;
```

Comment:
Also, i believe it's not necessary to initialize your size variable with a zero value. 

```suggestion

    private int size;
```